.github/workflows/release.yml

```diff
@@ -19,13 +19,14 @@ jobs:
         run: |
           REF=${{ github.ref }}
           echo ::set-output name=image_tag::$(echo $REF | sed 's/refs\/tags\/v//g')
+          echo ::set-output name=image_tag_minor_latest::$(echo $REF | sed -e 's/refs\/tags\/v//g' -e 's/\.[[:digit:]]$/-latest/g') 
```

Comment:
@nmvalera please let me know if my understanding is correct here:

| | Git Tag | Docker Tags |
| --- | ---- | ----| 
| 1 | `v20.10` | `20.10`|
| 2 | `v20.10.1` | `20.10.1` and `20.10-latest` |

In case we have more than 10 micro/patch releases in October, the Git Tag would be `20.10.11`, then the Docker Image tag for the latest should be `20.10-latest` but from the above regex, it would be `20.10.11-latest`.

Reply:
@nmvalera my suggestion for the latest tag is as below:
`echo ::set-output name=image_tag_minor_latest::$(echo $REF | sed -e 's/refs\/tags\/v//g' -e 's/^\(.....\).*/\1/' -e 's/$/-latest/'`
This will ensure we always get `20.10-latest` as the tag for minor/patch releases.

Reply:
Thanks a lot.

After rediscussing this the target is actually to set a `YY.MM` docker tag and not `YY.MM-latest`.

I pushed an update

- to update tag format to `YY.MM`
- fix the issue you mentioned

Reply:
@trung 

What we want to achieve is 


Docker tag | git | Comment
-- | -- | --
YY.MM.Patch | tag YY.MM.Patch | Image is built when pushing git tag YY.MM.Patch and then it is never overridden
YY.MM | latest tag YY.MM.* | Image is built for the first time when pushing git tag YY.MM.0 then it is overridden on every new YY.MM.* git tagOnce bumping to YY’.MM’.0 image is not overridden anymore